tools/publish

```diff
@@ -57,6 +67,8 @@ echo "-----> BUMP VERSION"
 echo ""
 
 npm version $VERSION_ARGS || exit 1
+
+npm run generate-docs
```

Comment:
this still won't be included as part of the commit. 

`npm version` creates a commit and tag:

> If run in a git repo, it will also create a version commit and tag. This behavior is controlled by git-tag-version (see below), and can be disabled on the command line by running npm --no-git-tag-version version. It will fail if the working directory is not clean, unless the -f or --force flag is set.

So the docs generation is outside of the bump commit. 

To keep this simple, we might be able to just do the docs generation after the `git push`, and commit/push that separately. It won't be included as part of the tag, but at least we can move forward with this PR and unlock the functionality.

Reply:
Seems good